src/engine/source/kvdb2/include/kvdb2/iKVDBHandler.hpp

```diff
@@ -16,8 +19,11 @@ class IKVDBHandler
     virtual std::variant<bool, base::Error> remove(const std::string& key) = 0;
     virtual std::variant<bool, base::Error> contains(const std::string& key) = 0;
     virtual std::variant<std::string, base::Error> get(const std::string& key) = 0;
+    virtual std::variant<std::unordered_map<std::string, std::string>, base::Error> dump() = 0;
 };
 
+using KVDBHandler = std::unique_ptr<IKVDBHandler>;
```

Comment:
I am not sure if this should be defined here, according to other modules criteria

Reply:
The same goes for all the occurrences